src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const i in robot) {
+    if (result[robot[i]]) {
```

Comment:
can use hasOwnProperty, but honestly, i don`t even know, if it makes some sense)